docs/source/CustomizingTheWorkflow/DASystem.rst

```diff
@@ -549,7 +555,7 @@ Snow depth observations are taken from the `Global Historical Climatology Networ
 
 where ``${YYYY}`` should be replaced with the year of interest. Note that these yearly tarballs contain all measurement types from the daily GHCN output, and thus, snow depth must be manually extracted from this broader data set.
 
-These raw snow depth observations need to be converted into IODA-formatted netCDF files for ingestion into the JEDI LETKF system. However, this process was preemptively handled outside of the Land DA workflow, and the initial GHCN IODA files for 2016 and 2020 were provided by NOAA PSL (Clara Draper).
+These raw snow depth observations need to be converted into IODA-formatted netCDF files for ingestion into the JEDI LETKF system. However, this process was preemptively handled outside of the Land DA workflow, and the 2019 GHCN IODA file was provided by NOAA PSL (Clara Draper).
 
 The IODA-formatted GHCN files are structured as follows (using 20160102 as an example):
```

Comment:
@gspetro-NOAA please correct the date.

Reply:
Oh, the 2016 file is just an example. I kept the examples because the year/date shouldn't matter.